Source/WebCore/page/ResizeObservation.cpp

```diff
@@ -52,24 +52,26 @@ void ResizeObservation::updateObservationSize(const BoxSizes& boxSizes)
     m_lastObservationSizes = boxSizes;
 }
 
-auto ResizeObservation::computeObservedSizes() const -> BoxSizes
+std::optional<ResizeObservation::BoxSizes> ResizeObservation::computeObservedSizes() const
 {
     if (m_target->isSVGElement()) {
         if (auto svgRect = downcast<SVGElement>(*m_target).getBoundingBox()) {
             auto size = LayoutSize(svgRect->width(), svgRect->height());
-            return { size, size, size };
+            return BoxSizes { size, size, size };
         }
     }
     auto* box = m_target->renderBox();
     if (box) {
-        return {
+        if (box->isSkippedContent())
+            return std::nullopt;
```

Comment:
Antti told me elsewhere that we use `{ }` in cases like this. Of course, here it is subtle and risky to do that!

Reply:
Right, I kept it as-is.